### PR TITLE
fs_watcher：对write的系统调用，进一步探测，更改挂载点的执行函数

### PR DESCRIPTION
**Pull request by sf1999817**
# Pull Request Template

## Description/描述

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
此 pr 解决了什么问题

Fixes # (issue)

## Type of change /PR 类型

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?/ 测试方式

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist/检查列表

- [ ] My code follows the style guidelines of this project /代码规范检查
- [ ] I have performed a self-review of my own code / 自己已经检查代码
- [ ] I have commented my code, particularly in hard-to-understand areas / 有代码注释，尤其是关键代码
- [ ] I have made corresponding changes to the documentation /变更内容同步到了文档中
- [ ] My changes generate no new warnings / 变更没有新产生告警
- [ ] I have added tests that prove my fix is effective or that my feature works /对修改代码添加了测试用例
- [ ] New and existing unit tests pass locally with my changes / 对单元测试也同步更新了

### CODE REVIEW

**Comment by helight** [comment]
@13186379707 review 一下

File: eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/include/stdlib.h
Comment (chenamy2017) [inline eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/include/stdlib.h:1]: 为什么提交标准库文件？

File: eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/write.c [hunk 1]
```diff
@@ -25,19 +25,18 @@ static int write_event(void *ctx, void *data, size_t data_sz)
...
 
 int main(int argc, char **argv)
 {
     struct ring_buffer *rb = NULL;
     struct write_bpf *skel;
-	int err;
+	  int err;
    
     /* Set up libbpf errors and debug info callback */
 	libbpf_set_print(libbpf_print_fn);
```
Comment (chenamy2017) [inline eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/write.c:42]: 修改下代码格式，比如基本的对齐。

**Comment by chenamy2017** [comment]
运行截图提供下。

File: eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/write.c (hunk 1 above)
Comment (sf1999817) [reply eBPF_Supermarket/Filesystem_Subsystem/fs_watcher/write.c:42]: 好的，师兄，我之前都是把代码运行截图放在了issues里面，对不起，我以后都放在pr里面。
捕获

**Comment by sf1999817** [comment]
微信图片_20240507225955

**Comment by sf1999817** [comment]
捕获

**Comment by 13186379707** [comment]
你要不再把你这三个文件的格式再调整一下，有的地方缩进是2个空格，有的地方缩进是4个空格
e1ef3894768431f9fafdc582d58a5d6
还有这些注释，基本和下一行的代码保持一致

**Pull request closed without merging**